cookbook/security/voters.rst

```diff
@@ -133,6 +134,13 @@ edit a particular object. Here's an example implementation::
                 return VoterInterface::ACCESS_DENIED;
             }
 
+            // double-check that the User object is the expected entity.
```

Comment:
no comment in this code block starts with a capitialized word. Let's fix this inconsistenty in another PR if we really care, ok?